Source/Core/AudioCommon/Mixer.cpp

```diff
@@ -52,6 +52,7 @@ void Mixer::DoState(PointerWrap& p)
   m_dma_mixer.DoState(p);
   m_streaming_mixer.DoState(p);
   m_wiimote_speaker_mixer.DoState(p);
+  m_skylander_portal_mixer.DoState(p);
```

Comment:
Don't forget to update the state version in State.cpp:
https://github.com/dolphin-emu/dolphin/blob/5d07a452410b84672c36eb5a26018f583121c164/Source/Core/Core/State.cpp#L99